Makefile.am

```diff
@@ -25,6 +25,7 @@ dist_systemdunit_DATA = \
 	eos-live-boot-overlayfs-setup.service \
 	eos-ostree-bootloader-setup.service \
 	eos-prune-printers.service \
+	eos-secondboot.service \
```

Comment:
This is a draft only because I don't like the name of the unit!

Reply:
Suggestions welcome for a better name that is accurate and not incredibly long...